.chloggen/tail_sampling_phases.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: connector/spanmetrics, processor/tail_sampling
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "Feature to support pre and post tail sampling phases, so other processors or connectors could see which traces will be exported."
```

Comment:
Does this need an update to reflect the latest changes?

Reply:
I think it makes senses to split the PR into two, or provide two updates into the change log.